js/project-list.js

```diff
@@ -0,0 +1,58 @@
+/* Project Card */
+
+var ProjectList = {
+  template: _.template($("script#project-list-template").html()),
+  buildHTML: function(projectData) {
+    var id = projectData.id;
+    var timestamp = new Date(projectData.Timestamp);
+    var month = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"][timestamp.getMonth()];
+    var issues = projectData.Issues ? projectData.Issues : "";
+    var issuesAsArray = issues.split(", ");
+    var dataForTemplate = {
```

Comment:
We should only pass what the `project-list-template` (to be renamed to `project-list-item-template`) in `index.html` needs.

i.e.,
`starred`,
`id`,
`title`,
`detailViewLink`,
`creators`,
`timestamp`